widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -5485,6 +5489,12 @@ div.prompt-panel .pentaho-toggle-button-container > DIV {
   padding: 0 0 4px 0;
 }
 
+#file-input-form {
+  display: flex;
+  position: absolute;
+  right: 24%;
+}
+
```

Comment:
This configuration looks strange, to me. To align this element to the right, you can configure the parent element instead with the following two properties:
```css
.file-input-form-container {
   display: flex;
   justify-content: end;
}
```